models/concerns/validate_data.rb

```diff
@@ -203,17 +203,23 @@ def check_position_relation_specificities
         @hash[:services].find{ |service| service[:id] == linked_id }
       }
       previous_service = nil
+      previous_activities = []
       services.each{ |service|
-        if previous_service && forbidden_position_pairs.include?([previous_service[:activity][:position], service[:activity][:position]])
-          inconsistent_position_services << [previous_service[:id], service[:id]]
-        end
+        activities = service[:activity] && [service[:activity]] || service[:activities]
+
+        previous_activities.each{ |previous_activity|
+          activities.each{ |activity|
+            inconsistent_position_services << [previous_service[:id], service[:id]] if forbidden_position_pairs.include?([previous_activity[:position], activity[:position]])
```

Comment:
line too long, rubocop doesn't detect because this cop is probably deactivated on this file :(